packages/core/src/components/ChoiceList/_ChoiceList.docs.scss

```diff
@@ -11,13 +11,19 @@ Style guide: components.choice
 */
 
 /*
```

Comment:
These changes right here look like merge conflicts, they should be safe to delete